### PR TITLE
Add fields.yml for apache module

### PR DESCRIPTION
**Pull request by ruflin**
Better descriptions of each field are still needed. But I don't have the background to do it 100% accurate. It is important that the template is now completed with the apache fields.

### CODE REVIEW

File: metricbeat/metricbeat.template.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
             "mapping": {
               "doc_values": true,
               "ignore_above": 1024,
-              "index": true,
+              "index": "not_analyzed",
```
Comment (tsg) [inline metricbeat/metricbeat.template.json]: Was this change made intentionally? I believe that's not valid for ES 5.0.

Comment (ruflin) [inline metricbeat/metricbeat.template.json]: No, this was generated automatically. I fixed it now by updating the base file.

File: metricbeat/etc/fields.yml [hunk 2]
```diff
@@ -1,14 +1,15 @@
 version: 1.0
 
 defaults:
-  type: keyword
```
Comment (ruflin) [inline metricbeat/etc/fields.yml]: @tsg Should should also be now keyword, and not string?

Comment (ruflin) [inline metricbeat/etc/fields.yml]: @tsg Found it in other beats and also fixed it.

**Comment by radoondas** [comment]
@ruflin I could help with description later if needed.

**Comment by tsg** [comment]
LGTM, merging like this, improvements into descriptions can come in later PRs (thanks @radoondas).

**Pull request merged**